.github/workflows/ci.yaml

```diff
@@ -81,7 +81,11 @@ jobs:
           registry-url: https://registry.npmjs.org/
       - run: node --version
       - run: npm ci
-      - run: npm run rebuild-docker-image
       - run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  deploy:
```

Comment:
Would slightly prefer another name here for this job, as it doesn't "deploy" Auspice. Perhaps `publish-image` or `rebuild-image`?

Reply:
Agreed, changed to `rebuild-docker-image`. I used `deploy` to be consistent with [Augur](https://github.com/nextstrain/augur/blob/9cd6eb489290536f8b9ad65c6d6c27843f19ca15/.github/workflows/ci.yaml#L52), but will update the name there in another PR.